nuxeo-opencmis-impl/src/main/java/org/nuxeo/ecm/core/opencmis/impl/server/CMISQLtoNXQL.java

```diff
@@ -301,6 +303,12 @@ public IterableQueryResult getIterableQueryResult(IterableQueryResult it, NuxeoC
         return new NXQLtoCMISIterableQueryResult(it, realColumns, virtualColumns, service);
     }
 
+    public PartialList<Map<String, Serializable>> getPartialList(PartialList<Map<String, Serializable>> pl,
```

Comment:
The name `getIterableQueryResult` was badly chosen  (but let's not change it right now as this has to be backported).
Let's rename the new method still please, `convertToCMIS` seems good?